src/components/Wizard/wizard.scss

```diff
@@ -86,14 +85,33 @@
 /* NewEditImportPane specific components */
 
 .new-form .form-buttons,
-.import-form .form-buttons {
+.import-form .form-buttons,
+.edit-form .form-buttons {
     margin-top: 1rem;
 
     .btn {
         @extend .col-2;
     }
 }
 
+.edit-form {
+    .edit-entry {
+        display: flex;
+    }
+
+    .form-check {
+        align-self: center;
+        width: 10%;
+    }
+
+    .btn-del {
```

Comment:
There is only one button here so we can just use `.btn` and we can omit the custom class names.